outputs.tf

```diff
@@ -6,6 +6,10 @@ output "subscription_id" {
   value       = local.resolved_subscription_id
   description = "Subscription id of log analytics Workspace "
 }
+output "resource_group_name" {
```

Comment:
what do you need it for? we usually just use the ID everywhere

Reply:
https://github.com/hmcts/rdo-terraform-hub-dmz/pull/219/files#diff-5071e4c1740dc041ad3121f7aa6941037eca93e174c196291ec30499ae6441f9R66

...unless you have a better way to do that?

Reply:
yeah commented there